### PR TITLE
Throw an error if invalid options are passed

### PR DESCRIPTION
**Pull request by baffo32**
It's a good thing I make so many mistakes, so I can write checks to detect them.

Ready for merge.

### CODE REVIEW

File: test/openBCIBoard-Impedance-test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('#impedanceTesting', function () {
   before(function (done) {
     ourBoard = new openBCIBoard.OpenBCIBoard({
       verbose: true,
-      fragmentation: k.OBCISimulatorFragmentationRandom
+      simulatorFragmentation: k.OBCISimulatorFragmentationRandom
```
Comment (aj-ptw) [inline test/openBCIBoard-Impedance-test.js]: good catch, did i miss this too?

**Comment by aj-ptw** [comment]
Please update `changeLog.md` with new name. Please code indicators (back ticks) in the change log so the line would be:

Three new initialization options: `simulatorFragmentation` , `simulatorBufferSize`, and `simulatorLatencyTimer`. Together, these new options enable a more _realistic_ simulator, mimicking different potential user computer systems.

something like that please, so people can read the change log and know where to look. Thanks! Once again love the new features flowing in!

**Comment by baffo32** [comment]
changelog updated !

File: test/openBCIBoard-Impedance-test.js (hunk 1 above)
Comment (aj-ptw) [inline test/openBCIBoard-Impedance-test.js]: smh... happy we merged with a staging branch then

**Pull request merged**